packages/react-query/vitest.config.ts

```diff
@@ -0,0 +1,13 @@
+import { defineConfig } from 'vitest/config'
+
+export default defineConfig({
+  test: {
+    name: '@suspensive/react-query',
+    dir: './src',
+    watch: false,
+    environment: 'jsdom',
+    globals: true,
+    setupFiles: './test.setup.ts',
+    coverage: { provider: 'v8' },
+  },
```

Comment:
we don't have to use alias?!

I think It could possible needed

```tsx
 alias: {
      '~': '/src',
      
    },

```

Reply:
You mean [this `alias` api](https://vitest.dev/config/#alias). I didn't know it.
I will try to know it. wait a moment